packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2697,20 +2696,18 @@ export interface FileDownloadError {
 }
 
 // @beta
-export type FileDownloadHandler = (userId: string, fileMetadata: FileMetadata) => Promise<URL | FileDownloadError>;
+export type FileDownloadHandler = (userId: string, fileMetadata: AttachmentMetadata) => Promise<URL | FileDownloadError>;
 
 // @beta
-export type FileMetadata = FileSharingMetadata | /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ ImageFileMetadata;
-
-// @beta (undocumented)
-export type FileMetadataAttachmentType = 'fileSharing' | /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ 'inlineImage' | 'unknown';
-
-// @beta
-export interface FileSharingMetadata extends BaseFileMetadata {
+export interface FileMetadata {
     // (undocumented)
-    attachmentType: 'fileSharing';
+    attachmentType: 'file';
+    extension: string;
+    id: string;
+    name: string;
     // (undocumented)
```

Comment:
Optional: Would be nice to add document of what this payload is about and what are the keys and values.

Reply:
The type is documented: https://github.com/Azure/communication-ui-library/pull/3846/files/0c588a84d91b17d2c283c66bb7bc83abfc6a1bcb#diff-d257e524dc363b5145a6b1c8b21f6f13fa9238b80aa0e665ce5505edc58ee9d9R29
I'm not sure why it's not coming through at this level with the tooling,.